docs/evm/networks.md

```diff
@@ -11,32 +11,32 @@ EVM on Flow has the following public RPC nodes available:
 # Previewnet
 
 | Name            | Value                                   |
-|-----------------|-----------------------------------------|
+| --------------- | --------------------------------------- |
 | Network Name    | Previewnet                              |
 | Description     | The public RPC URL for Flow Previewnet  |
 | RPC Endpoint    | https://previewnet.evm.nodes.onflow.org |
 | Chain ID        | 646                                     |
 | Currency Symbol | FLOW                                    |
 | Block Explorer  | https://previewnet.flowdiver.io         |
 
-# Testnet (Coming Soon)
-
-| Name            | Value                                  |
-|-----------------|----------------------------------------|
-| Network Name    | Testnet                                |
-| Description     | The public RPC URL for Flow Testnet         |
-| RPC Endpoint    | https://testnet.evm.nodes.onflow.org   |
-| Chain ID        | 545                                    |
-| Currency Symbol | FLOW                                   |
-| Block Explorer  | https://testnet.flowdiver.io           |
-
-# Mainnet (Coming Soon)
-
-| Name            | Value                                  |
-|-----------------|----------------------------------------|
-| Network Name    | Mainnet                                |
-| Description     | The public RPC URL for Flow Mainnet    |
-| RPC Endpoint    | https://mainnet.evm.nodes.onflow.org   |
-| Chain ID        | 747                                    |
-| Currency Symbol | FLOW                                   |
-| Block Explorer  | https://flowdiver.io                   |
+# Testnet
+
+| Name            | Value                                |
+| --------------- | ------------------------------------ |
+| Network Name    | Testnet                              |
+| Description     | The public RPC URL for Flow Testnet  |
+| RPC Endpoint    | https://testnet.evm.nodes.onflow.org |
+| Chain ID        | 545                                  |
+| Currency Symbol | FLOW                                 |
+| Block Explorer  | https://testnet.flowdiver.io         |
```

Comment:
```suggestion
| Block Explorer  | https://evm-testnet.flowscan.io         |
```
This naming is confirmed by @bjartek and @SeanRobb

Reply:
updated!